spec/models/quiz_spec.rb

```diff
@@ -62,4 +62,27 @@
       expect(quiz.student_score(user)).to eq(1)
     end
   end
+
+  describe "#first_unanswered_question" do
+    it "returns a student's first unanswered question" do
+      first_question = quiz.questions.first
+      FactoryGirl.create(:answer_submission,
+        user: user,
+        answer: first_question.correct_answer
+      )
+      second_question = FactoryGirl.create(:question_with_answers, quiz: quiz)
+
+      expect(quiz.first_unanswered_question(user)).to eq(second_question)
+    end
+
+    it "returns nil if student has answered all questions" do
+      first_question = quiz.questions.first
+      FactoryGirl.create(:answer_submission,
+      user: user,
+      answer: first_question.correct_answer
```

Comment:
I added launchcop to the gemfile. After you get your tests passing, and before you push your code to github, get into the habit of running `bundle exec launchcop` and fixing any errors you might find.

The message you will get from this line is: "spec/models/quiz_spec.rb:81:7: C: Align the parameters of a method call if they span more than one line." It should say to indent the lines like on 71, 72. Which is what the launchcop settings are set to. This seems like a rubocop bug.